server/src/test/app.d/02-python.py

```diff
@@ -23,7 +23,7 @@ def demo_implicit():
   global world_imp; world_imp = TableTools.timeTable("00:00:01")
 
 def initialize_implicitly(func: Callable[[], None]):
-  app = jpy.get_type("io.deephaven.grpc_api.appmode.ApplicationContext").get()
+  app = jpy.get_type("io.deephaven.appmode.ApplicationContext").get()
```

Comment:
Odd that this isn't failing in main right now, I already checked in this change. Will validate that these tests actually can fail...

Reply:
@devinrsmith can you confirm that these tests don't actually care what is in these files, as long as they exist? That seems to be the case, I can mangle their contents without failing any tests

Reply:
It looks like we have tests for them, but they are ignored.

See `io.deephaven.grpc_api.appmode.ApplicationTest#app02`.

Reply:
Do we have an issue for that? As this reflects, this seems subject to bitrot

Reply:
Doesn't look like it...